### PR TITLE
[DNM]Add leveldb and rocksdb for timeseries

### PR DESCRIPTION
**Pull request by liukun4515**
Install the TsLeveldb

https://github.com/liukun4515/TSLevelDB

branch ： java_port

### CODE REVIEW

**Comment by liukun4515** [comment]
PTAL @liuruiyiyang @little-emotion

File: src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/kvdb/TimeSeriesKVDB.java [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+        org.rocksdb.Options optionsRocks = new org.rocksdb.Options();
+        optionsRocks.setCreateIfMissing(true);
+        iTimeSeriesDB = RocksDBTimeSeriesDBFactory.getInstance().openOrCreate(file, optionsRocks);
+        break;
+    }
+    return iTimeSeriesDB;
+  }
+
+  @Override
+  public void init() {
```
Comment (liuruiyiyang) [inline src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/kvdb/TimeSeriesKVDB.java]: throws TsdbExceptions?

File: src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/kvdb/TimeSeriesKVDB.java [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  @Override
+  public void close() {
+    // Not need to implement
+    synchronized (TimeSeriesKVDB.class) {
+      if (reference == 1) {
+        try {
+          this.timeSeriesDB.close();
+          this.timeSeriesDB = null;
+        } catch (IOException e) {
+          // TODO throws exception
```
Comment (liuruiyiyang) [inline src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/kvdb/TimeSeriesKVDB.java]: throws TsdbException

File: src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/kvdb/TimeSeriesKVDB.java [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+  @Override
+  public Status insertOneBatch(Batch batch) {
+    try {
+      ITimeSeriesWriteBatch timeSeriesWriteBatch = timeSeriesDB.createBatch();
+      for (Entry<Long, List<String>> entry : batch.getRecords().entrySet()) {
+        for (int i = 0; i < entry.getValue().size(); i++) {
+          String timeseries =
+              batch.getDeviceSchema().getDevicePath() + "." + batch.getDeviceSchema().getSensors()
+                  .get(i);
+          long time = entry.getKey();
```
Comment (liuruiyiyang) [inline src/main/java/cn/edu/tsinghua/iotdb/benchmark/tsdb/kvdb/TimeSeriesKVDB.java]: long time = entry.getKey();这行应该写到上层循环里面吧

**Pull request closed without merging**